libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts

```diff
@@ -411,6 +411,7 @@ export class SkyAgGridDataManagerAdapterDirective
   }
 
   #applyColumnWidths(): void {
+    /* istanbul ignore if: safety check */
```

Comment:
See comment from John's PR a while ago (this condition was added for tests): https://github.com/blackbaud/skyux/pull/2681/files#r1745689224